File: src/main/java/Launcher.java
import Logic.Save;
import UI.MainWindow;
import java.io.File;

public class Launcher {
    public static void main(String[] args) {
        File saveDir = new File(Save.SAVE_DIR);
        if (!saveDir.exists()) {
            saveDir.mkdirs();
        }

        new MainWindow();
    }
}

File: src/main/kotlin/Launcher.kt
import Logic.Save
import UI.MainWindow
import java.io.File

object Launcher {
    @JvmStatic
    fun main(args: Array<String>) {
        val saveDir = File(Save.SAVE_DIR)
        if (!saveDir.exists()) {
            saveDir.mkdirs()
        }
        MainWindow()
    }
}
